packages/common/package.json

```diff
@@ -41,6 +41,7 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
+    "@ethereumjs/common": "2.6.2",
```

Comment:
I like moving the getCommon functionality out of the HD wallets, I am a little concerned with adding a ~2.5mb package to common

Reply:
I think it's okay since it won't have any effect on bundle size and is dynamically imported so won't effect users unless they use the hdwallets